README.md

```diff
@@ -2,4 +2,4 @@
 
 ## Versions
 
-- [v0.1](versions/0.1.md) - *Under Development*
+- [v0.1](versions/0.1.md) - *Current*
```

Comment:
In the spirit of conventional commits, let's just land this as `v1.0.0` (or if you really want to keep this pre-1.0.0, 0.1.0).

Reply:
I saw you landed `v0.1` already, perhaps just make this `v0.0.1`? and the next version can be `v1.0.0` (just thinking out loud).